### ISSUE TITLE
Stack record is missing version in CRD details

### ISSUE THREAD

**Issue by displague** [crossplane/app-wordpress#18]
@thephred observed and I confirmed, that the sample-stack-wordpress, when installed has an odd CRD listing in the Stack record:

```
    customresourcedefinitions:
    - apiVersion: wordpress.samples.stacks.crossplane.io/
      kind: WordpressInstance
    dependsOn:
    - crd: kubernetesclusters.compute.crossplane.io/v1alpha1
    - crd: mysqlinstances.database.crossplane.io/v1alpha1
    - crd: kubernetesapplications.workload.crossplane.io/v1alpha1
```

The expected behavior is that the version would be shown in the `apiVersion` line, as it does for stack-gcp and other stacks.

I'm not sure if this is a stack-manager or sample-stack-wordpress bug.

**Comment by suskin** [comment]
@displague @thephred what is the impact of this issue?

For what it's worth, the wordpress sample stack does not appear to contain any sort of `customresourcedefinitions` block in its yaml. In `master`, anyway . . .

This is a bit of a tangent now, but maybe it'd make sense to set up a job to continuously publish the stack when the repository is changed.

**Comment by displague** [comment]
> For what it's worth, the wordpress sample stack does not appear to contain any sort of customresourcedefinitions block in its yaml. In master, anyway

The `app.yaml` wouldn't contain a `customresourcedefinitions` block, the `Stack` resource's `spec` would.  Is that what you observed?

**Comment by displague** [comment]
@thephred This appears to be fixed in the latest `master` builds (tested locally).

```
  customresourcedefinitions:
  - apiVersion: wordpress.samples.stacks.crossplane.io/v1alpha1
    kind: WordpressInstance
  dependsOn:
  - crd: kubernetesclusters.compute.crossplane.io/v1alpha1
  - crd: mysqlinstances.database.crossplane.io/v1alpha1
  - crd: kubernetesapplications.workload.crossplane.io/v1alpha1
```

This should be resolved when we publish a new tagged release.

**Comment by displague** [comment]
Confirmed to be fixed in v0.7.0 (although I think this was fixed in the v0.6.0 release).
